app/User.php

```diff
@@ -433,6 +436,12 @@ public function isFrauding(): bool
         if (!empty($this->provider)) {
             return false;
         }
+
+        $validation = UserValidation::where('user_id', $this->id)->where('is_valid', true)->first();
```

Comment:
Since we are already on the user, it would be simpler to make it a relation on UserRelations , and is_valid into a scope `$this->userValidation->valid()->first()`